src/extension.ts

```diff
@@ -109,7 +128,7 @@ function executeAmebaOnWorkspace(ameba: Ameba | null) {
     if (!ameba) return;
 
     for (const doc of workspace.textDocuments) {
-        if (isValidCrystalDocument(doc)) {
+        if (isCrystalDocument(doc)) {
             outputChannel.appendLine(`[Workspace] Running ameba on ${getRelativePath(doc)}`);
             ameba.execute(doc);
         }
```

Comment:
Shouldn't this check for virtual documents as well?

Reply:
It does